source/io/module_eggShell.js

```diff
@@ -479,6 +483,27 @@
             }
         };
 
+        Module.eggShell.writeStringFromArray = function (valueRef, inputArray) {
```

Comment:
Talked with memo and instead we will create a `writeTypedArray` function that is more generic for writing arbitrary TypedArrays to memory and that it will support String types.

Will include a comment saying that it can be expanded for more types in the future as needed